src/client/driver/status.js

```diff
@@ -4,6 +4,7 @@ export default class DriverStatus extends Assignable {
     constructor (obj) {
         super(obj);
 
+        this.id              = Date.now();
```

Comment:
I see a potential issue here. If i write on my page
```js
window.performance = () => { return 0; // maybe implement here some custom logic }
```
our id generation is failed. 
We have `hammerhead.nativeMethods.dateNow` (because we've found the page where `Date` was overriden) but don't have the same for `performance.now` and `Math.random`. I suppose we should do it in hammerhead? /cc @inikulin @churkin